metrics/New_Contributor_Closing_Issues.md

```diff
@@ -1,27 +1,28 @@
 # New Contributors Closing Issues
 
-Question: How many contributors are closing issues for the first time?
+Question: How many contributors are closing issues for the first time in a given project?
```

Comment:
Proposing to use the phrase "in a given repository" instead of "in a given project", as a "project" often contains multiple repositories.

Reply:
You can make this change if you want, but I think this is more of a technical discussion between "project" and "repository". The same remark also holds the other way around, since I know of many git "repositories" that host multiple "projects". (Typically, in the repository you will have some kind of folder structure to separate the projects.) Of course it is also true that a single project can contain multiple repositories.
Issues could be addressed through a separate issue tracking system such as BugZilla or alternatives (as opposed to through GitHub directly). In that case, the issues could be linked to the project (or one of its subproject) without necessarily needing to be aware of the repositories on GitHub that correspond to this project.